connect-inject/handler_test.go

```diff
@@ -97,6 +97,10 @@ func TestHandlerHandle(t *testing.T) {
 					Operation: "add",
 					Path:      "/spec/initContainers",
 				},
+				{
+					Operation: "add",
+					Path:      "/spec/initContainers/-",
```

Comment:
What's the difference between L102 and L98? Do you know what determines if you need a - or not? Can we use the - for both?

Reply:
The slash is for when there is more than one entry from the patch operation. So if there was a single entry it would just be `/spec/initContainer` and if there were more than one, the additional ones would have the `/-`.
Does that make sense?

Reply:
I see, thanks!!!